docs/operator/vars.MD

```diff
@@ -0,0 +1,114 @@
+---
+sort: 13
+---
+
+# Auto Generated vars for package config 
+## updated at Tue Oct 12 07:44:08 UTC 2021 
```

Comment:
Not sure if it needs to be a header - it makes navigation awkward:
![image](https://user-images.githubusercontent.com/2902918/148677001-e0da6d39-a6a3-4d17-978b-a7ade276698f.png)

Reply:
Can you suggest formatting style?

Reply:
I'd suggest to not use `#` mark for `updated at...` info, just make it bold/italic or leave it without formatting